config.json

```diff
@@ -1464,6 +1464,19 @@
       "topics": [
         "graph_theory"
       ]
+    },
```

Comment:
I think this file is _ordered_. Or at least it was. It should be around the other `unlocked_by` `simple-cipher`, and then it's probably sorted by difficulty > lexographically.